Makefile

```
@@ -293,6 +293,10 @@ kind-tekton:
 kind:
 	./hack/install-kind.sh
 	./hack/install-registry.sh
+verify:
```

We are not consistent with this yet, but I would like to start to introduce [`.PHONY`](https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html) with new Makefile targets. Later, we should make sure we have the `.PHONY` wherever it applies. 
```suggestion

.PHONY: verify
verify:
```